sycl/include/CL/sycl/aspects.hpp

```diff
@@ -38,7 +38,9 @@ enum class aspect {
   ext_intel_gpu_subslices_per_slice = 22,
   ext_intel_gpu_eu_count_per_subslice = 23,
   ext_intel_max_mem_bandwidth = 24,
-  ext_intel_mem_channel = 25
+  ext_intel_mem_channel = 25,
+  usm_atomic_host_allocations = 26,
+  usm_atomic_shared_allocations = 27
```

Comment:
NIT: suggest finishing this line with a comma.
```suggestion
  usm_atomic_shared_allocations = 27,
```